test/sharness/t0060-daemon.sh

```diff
@@ -14,6 +14,9 @@ test_expect_success "setup IPFS_PATH" '
 '
 
 # NOTE: this should remove bootstrap peers (needs a flag)
+# TODO(cryptix):
+#  - we won't see daemon startup failure because we put the daemon in the background - fix: fork with exit code after api listen
+#  - also default ports: might clash with local clients. Failure in that case isn't clear as well because pollEndpoint just uses the already running node 
```

Comment:
I think, this is what drove us in circles in #1277 - On dev box, there might be another daemon running (on test env there might be an unkilled daemon still running). The effect is that pollEndpoint passes by dialing another daemon.